apps/BraynsViewer/BraynsViewer.cpp

```diff
@@ -130,9 +130,11 @@ void BraynsViewer::display( )
 
     std::stringstream ss;
     ss << "Brayns Viewer - Interactive Ray-Tracing";
-    size_t ts = _brayns->getParametersManager().getSceneParameters().getTimestamp();
-    if( ts != std::numeric_limits<size_t>::max() )
+    float ts = _brayns->getParametersManager().getSceneParameters().getTimestamp();
+    if( ts != std::numeric_limits< float >::max() )
         ss << " (frame " << ts << ")";
```

Comment:
I like the timestamp as a float, rather than a frame as a int. In any simulation, what is interesting is the timestamp.